package.json

```diff
@@ -7,14 +7,14 @@
     "./src/crypto-sha1-2.js": "./src/crypto-sha1-2-browser.js"
   },
   "scripts": {
-    "test": "aegir-test",
-    "test:browser": "aegir-test browser",
+    "test": "aegir-test --webworker",
```

Comment:
The tests now run three times, once in node, once in the browser without webworkers and once inside webworkers